### ISSUE TITLE
SSL=0 still try to verify SSL cert

### ISSUE THREAD

**Issue by jinyangzhen** [narendrans/sqlalchemy_dremio#27]
I try to test the new driver without SSL, but looks not work. Is this expected?

```
>>> from sqlalchemy import *
>>> db_uri = "dremio+flight://dremio:dremio123@localhost:32010/dremio;SSL=0"
>>> engine = create_engine(db_uri)
>>> sql = 'SELECT * FROM sys.options limit 5 -- SQL Alchemy Flight Test '
>>>
>>> result = engine.execute(sql)
```

***************
['UID=dremio', 'PWD=dremio123', 'HOST=localhost', 'PORT=32010', 'Schema=dremio', 'SSL=0']
***************
E0922 22:19:30.986260000 4618952128 ssl_utils.cc:570]                  load_file: {"created":"@1663856370.986224000","description":"Failed to load file","file":"/Users/runner/work/crossbow/crossbow/vcpkg/buildtrees/grpc/src/17cc203898-db2679e7f2.clean/src/core/lib/iomgr/load_file.cc","file_line":72,"filename":"/usr/share/grpc/roots.pem","referenced_errors":[{"created":"@1663856370.986220000","description":"No such file or directory","errno":2,"file":"/Users/runner/work/crossbow/crossbow/vcpkg/buildtrees/grpc/src/17cc203898-db2679e7f2.clean/src/core/lib/iomgr/load_file.cc","file_line":45,"os_error":"No such file or directory","syscall":"fopen"}]}
E0922 22:19:30.986294000 4618952128 ssl_security_connector.cc:413]     Could not get default pem root certs.
E0922 22:19:30.986304000 4618952128 secure_channel_create.cc:108]      Failed to create secure subchannel for secure name 'localhost:32010'
E0922 22:19:30.986313000 4618952128 secure_channel_create.cc:50]       Failed to create channel args during subchannel creation.
E0922 22:19:30.986352000 4618952128 ssl_security_connector.cc:413]     Could not get default pem root certs.
E0922 22:19:30.986360000 4618952128 secure_channel_create.cc:108]      Failed to create secure subchannel for secure name 'localhost:32010'
E0922 22:19:30.986367000 4618952128 secure_channel_create.cc:50]       Failed to create channel args during subchannel creation.

**Comment by stepcheunghk** [comment]
@jinyangzhen I also found this issue. So I checked out the db.py file from sqlalchemy_dremio (which is downloaded by pip). I noticed that they are using _**grpc+tls**_ in the library.

`client = flight.FlightClient('grpc+tls://{0}:{1}'.format(properties['HOST'], properties['PORT']))`

However, when I checked out their code in master branch d41b873f8c45d0395c8f102f47ec0c432f38933c, I see they are using **_grpc+tcp_**

`client = flight.FlightClient('grpc+tcp://{0}:{1}'.format(properties['HOST'], properties['PORT']))`

It is really strange.

cc @narendrans

**Comment by jinyangzhen** [comment]
<img width="1736" alt="image" src="https://user-images.githubusercontent.com/1779918/194030587-756b21fc-3249-4f33-95bb-de2fd43f7ecb.png">
@stepcheunghk my local lib shows the grpc+tcp version. But i can work around this issue by disabling ssl verification in FlightClient as shown in above pic. 

From the code, it looks like SSL parameter hasn't been taken effect. It needs to be improved to honor this parameter in order to fully fix this issue.

**Comment by jduo** [comment]
Fixed in #29. The parameter is UseEncryption and defaults to true.
Note that you should use a '?' to after the schema instead of ';'. And use '&' to separate key-value pairs after the ?
